context.copytostorage/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="context.copytostorage" name="Copy File To Storage" version="1.0.0" provider-name="Abel Tesfaye">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="kodi.context.item">
+    <menu id="kodi.core.main">
+      <item library="addon.py">
+        <label>32000</label>
+        <visible>yes</visible>
```

Comment:
not sure visible yes is valid, maybe you mean visible true. It'd be nice if you could restrict the visibility to a more stricter condition. I don't think you can copy every single listitem to storage...